src/js/app.js

```diff
@@ -0,0 +1,46 @@
+class App {
+  #model;
+  #view;
+  constructor({ target, model, view }) {
+    this.$target = document.querySelector(target);
+    this.$amountInput = document.querySelector("#purchase-amount-input");
+    this.#model = model;
+    this.#view = view;
+    this.setEvent();
+  }
+
+  onSubmit(form) {
+    const submitHandlers = {
+      "purchase-input-form": (amount) => {
+        this.#model.purchaseLotto(amount);
+        this.render(this.#model.get());
```

Comment:
model에서 [getter](https://ko.javascript.info/property-accessors#ref-503)를 사용하는것도 좋아보이네요 ㅎㅎ
```suggestion
        this.render(this.#model.state);
```